app/javascript/react/components/MonsterForm.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+
+const MonsterForm = () => {
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+
+  const [formData, setFormData] = useState({
+    name: "",    description: "",
+    classification: "",
+    habitat: "",
+  })
+  const[postResponse, setPostResponse] = useState("")
```

Comment:
Ahh I see this is meant to be a notification! We can remove it and where it is being displayed since we are instead redirecting to another view, but I like the idea!  To provide immediate feedback we could redirect to the new show page, but that can be added in as a later update.